app/services/pul_metadata_services/client.rb

```diff
@@ -38,6 +38,9 @@ def retrieve_aspace_pulfa_ead(id)
       # @param source_metadata_id [String] the remote metadata identifier
       # @return [Boolean]
       def bibdata?(source_metadata_id)
+        # 3506421 will be in all alma IDs, and old Voyager records are
```

Comment:
We should update this comment. only 6421 are guaranteed precisely (although there will always be a 2-digit code at the beginning as well)

I just found this posted in slack by @mzelesky:

`99 Record type; 350 - Voyager source; 6421 - Princeton's Alma`